packages/redot-cli/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../tsconfig.json"
+}
```

Comment:
Doesn’t this need a `references` for TS to work?
It took me a bit to get it right but with this change the process becomes much faster than with the separate `tsc`s we used to have in places: https://github.com/remarkjs/remark/commit/f6bd64e6143ac8d01f731eb2e57e14bb0b10f1d1

Reply:
Thanks for the pointer,
It works, but it may not be as fast as it could be.
I aimed to match this with `remark`, but may have missed some config, I'll double check it.

Reply:
Basically if you add a reference for each dependency inside the workspace, the build can be significantly sped up. However, cleaning the cache before building works against this.

It could be even faster by making a TypeScript project (e.g. the `tsconfig.json` file in the workspace root) that excludes all files, but references all workspaces. Then instead of running `tsc` in each workspace, running `tsc` in the project root.